api-documentation/invoice-line.md

```diff
@@ -161,3 +161,83 @@ Invoice not found.
 * `paypal`
 * `external`
 {% endhint %}
+
+## Line types
+
+The invoice lines can be of 2 basic types: increasing the invoice balance (regular lines, fees, chargebacks) and will have a positive amount attribute or decreasing the invoice balance (payments, credits) and will have a negative amount attribute. Furthermore lines can be categorized by their type (e.g.: Installment fees, Late payment fees).
+
+The type of an invoice line is determined by the `type` attributes (always present) and the optional `subtype` attributes. Below is a list of possible values.
+
+
+### Regular Lines
+
+This line type is used for lines added to the invoice at creation time, when increasing the invoice amount
+from the ClubCollect web interface or when a member adds a donation to the invoice (if enabled).
+
+| Type | Subtype | Description |
+|-|-|-|
+| `INVOICE-LINE` | | Regular Invoice line |
+| `INVOICE-LINE` | `donation` | Donation Invoice line |
+| `INVOICE-LINE` | `surcharge` | Payment process fee Invoice line |
+
+### Fees Lines
+
+At certain points of the life cycle of an invoice fees can be added to the invoice (e.g. Member initiating a payment
+in installments).
+
+
+| Type | Subtype | Description |
+|-|-|-|
+| `INSTALLMENT-FEE` | | Fee members must pay when paying for an invoice in installments |
+| `CHARGEBACK-FEE` | | When a payment is charged back a fee will be added to the invoice |
+| `LATE-PAYMENT-FEE` | | Late payment fee added when reminders are sent |
+
+### Payment Lines
+
+When payments are made (either directly by member or external payments added from the Club Collect web interface)
```

Comment:
I would leave out any mention to "ClubCollect web interface" or to internal logic, trying to keep it simple and straight to the point

> Payment lines are added to the invoice to register new payments happening for this invoice

Reply:
```suggestion
Payment lines are added to register new payments happening for an invoice.
```